olive/model/hf_utils.py

```diff
@@ -44,8 +178,25 @@ def task_or_model_class_required(cls, v, values):
                 raise ValueError("Either task or model_class must be specified")
         return v
 
-
-def load_huggingface_model_from_task(task: str, name: str):
+    def load_model(self, model_path: str = None):
+        """Load model from model_path or model_name"""
+        # TODO: handle self.config, probably need to pass it as part of loading_args
+        model_name_or_path = model_path or self.model_name
+        loading_args = self.model_loading_args.get_loading_args() if self.model_loading_args else {}
+        if self.task:
+            model = load_huggingface_model_from_task(self.task, model_name_or_path, loading_args)
+        elif self.model_class:
+            model = load_huggingface_model_from_model_class(self.model_class, model_name_or_path, loading_args)
+        return model
+
+    def load_model_config(self, model_path: str = None):
+        """Load model config from model_path or model_name"""
+        # TODO: handle self.config, probably need to pass it as kwargs
+        model_name_or_path = model_path or self.model_name
+        return get_hf_model_config(model_name_or_path)
+
+
+def load_huggingface_model_from_task(task: str, name: str, kwargs: Dict[str, Any] = None):
```

Comment:
>kwargs [](http://example.com/codeflow?start=59&length=6)

add **?

Reply:
Yes, I can do that. I wasn't sure if we should except anything as `**kwargs` or have a dict the user to needs to pass.

Reply:
Updated to use **kwargs instead dict param.

Reply:
**kwargs is a dict actually

Reply:
Sorry, I meant instead of asking for a dict called kwargs args to be passed line `func(kwargs)`, they can pass it like regular kwargs `func(**kwargs)`.

Reply:
Actually, they are same regardless which way we use.